kolibri/plugins/coach/assets/src/routes/planLessonsRoutes.js

```diff
@@ -34,12 +35,25 @@ function path(...args) {
 
 const { showLessonsRootPage } = useLessons();
 
+function classIdParamRequiredGuard(toRoute, subtopicName) {
```

Comment:
I think you hinted at this on Slack - but yes, it would be better if this function were factored out and imported in each of these places. Rule of 3 applies (as I see it again below!)

Reply:
done!